### PR TITLE
update apiVersion p1

### PR DESCRIPTION
**Pull request by andreipope**
update apiVersion p1

### CODE REVIEW

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/logmanagement-efk-k8s.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ data:
        num_threads 8
     </match>
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (rodrigc) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/logmanagement-efk-k8s.md]: @andreipope `DaemonSet` is served from `apps/v1`, see: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#daemonset-v1-apps

File: content/portworx-install-with-kubernetes/on-premise/install-pks/install-pks-vsphere-local.md [hunk 2]
```diff
@@ -467,7 +467,7 @@ spec:
       port: 8099
       targetPort: 8099
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (rodrigc) [inline content/portworx-install-with-kubernetes/on-premise/install-pks/install-pks-vsphere-local.md]: Deployment should be served from `apps/v1`.  See: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#deployment-v1-apps

**Comment by rodrigc** [comment]
@andreipope you need to go back over your other PR's and make the equivalent changes.

Double-check what you changed with the API docs here: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/logmanagement-efk-k8s.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/logmanagement-efk-k8s.md]: @rodrigc Gotcha. Will double-check.

**Comment by rodrigc** [comment]
@andreipope as you are working on these fixes, you can validate the syntax of the yaml files that you are modifying with `kubectl`.  You don't need a running kubernetes cluster for this.  You can:

1.  Download a kubectl binary from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#downloads-for-v1160-alpha2
2.  If you modify a yaml file, such as `example.yaml`, then run:
```
kubectl apply --dry-run -f example.yaml
```

**Comment by andreipope** [comment]
> @andreipope as you are working on these fixes, you can validate the syntax of the yaml files that you are modifying with `kubectl`. You don't need a running kubernetes cluster for this. You can:
> 
> 1. Download a kubectl binary from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#downloads-for-v1160-alpha2
> 2. If you modify a yaml file, such as `example.yaml`, then run:
> 
> ```
> kubectl apply --dry-run -f example.yaml
> ```

Sure, that's a good idea!

**Pull request merged**